### ISSUE TITLE
StringValidate.Regex.showResult fails

### ISSUE THREAD

**Issue by SethTisue** [fthomas/refined#549]
in a clean checkout, if I do `coreJVM/testOnly *StringValidate*` I get:

```
[info] ! StringValidate.Regex.showResult: Falsified after 0 passed tests.
[info] > Labels of failing property: 
[info] Expected "Regex predicate failed: Unclosed group near index 4\u000a(a|b\u000a    ^" but got "Regex predicate failed: Unclosed group near index 4\u000a(a|b"
```

seen in the Scala 2.12 community build

**Comment by fthomas** [comment]
It seems that this test is sensitive to the JDK version. It is just fine with Oracle JDK 8 and I guess you and the community build use a different JDK, right?

Nevertheless, it should be fixed with #551.

**Comment by SethTisue** [comment]
I thought I had seen this on Oracle JDK 8, but it's possible I got confused about what Java version I was running.

regardless, your fix looks good. I'l come back if it crops up again
